mep/people/management/commands/export_creators.py

```diff
@@ -12,12 +12,29 @@
 
 
 class Command(ExportMemberCommand):
-    """Export member data."""
+    """Export creator data."""
+
+    csv_fields = [
+        "id",  # slug will be used (unlike members, no URI to use)
+        "name",
+        "sort_name",
+        "title",
+        "gender",
+        "is_organization",
```

Comment:
do we have any creator orgs?

Reply:
Looks like one:

```
In [2]: Person.objects.filter(creator__isnull=False, is_organization=True)
Out[2]: <PersonQuerySet [<Person pk:10524 Fabian Society>]>
```

Reply:
wow, fascinating; project full of edge cases